internal/pkg/describe/status_test.go

```diff
@@ -480,15 +477,15 @@ Last Deployment
 Task Status
 
   ID                Image Digest         Last Status         Started At          Stopped At          Health Status
-  12345678          69671a96,ca27a44e    RUNNING             14 years ago        14 years ago        HEALTHY
+  12345678          69671a96,ca27a44e    RUNNING             -                   -                   HEALTHY
```

Comment:
Oh do we have some code that handles this? in case of the default value it becomes "-"

Reply:
it's already in the ecs pkg